### DISCUSSION TITLE
web view is broken

### DISCUSSION THREAD

**Discussion by aliakseika** [openthread/openthread#9083] [Q&A]
Hello. There was such problem: I try to study openred now. I use the TI CC1352P1-4 device, I installed an application from the list of examples (tempsensor) on it. The otbr uses the latest version from the current repository. The device is added to the network, coap requests work, but the web display breaks, I can’t view either the network map or even the status. Maybe someone came across? I apologize if I described it rather chaotically, but I am writing through a translator. I can immediately answer any clarifying questions if I described something incomprehensibly. 
[screenshot](https://prnt.sc/MA0vuCgueiIf)
restarting the otbr-gent service does not help

**Comment by arnoldlv1976** [comment]
I worked with the latest version of web interface of the Border Router and it works properly. You can check on the Border Router side if the "otbr-web" service is on. "sudo systemctl status otbr-web" it should be active (running).

**Comment by aliakseika** [comment]
services are all running, only the network status page does not open. This happens after a CC1352 device with an example application tempsensor is added to the network
